### ISSUE TITLE
ImportError: cannot import name 'itkHelpers' from 'itk.support' (unknown location)

### ISSUE THREAD

**Issue by lucasjome** [InsightSoftwareConsortium/ITKTextureFeatures#91]
```
Traceback (most recent call last):
  File "/home/user/a.py", line 27, in <module>
    filtr = itk.CoocurrenceTextureFeaturesImageFilter.New(im)
  File "/home/user/py_env/lib/python3.7/site-packages/itk/support/lazy.py", line 76, in __getattribute__
    base.itk_load_swig_module(module, namespace)
  File "/home/user/py_env/lib/python3.7/site-packages/itk/support/base.py", line 110, in itk_load_swig_module
    l_module = loader.load(swig_module_name)
  File "/home/user/py_env/lib/python3.7/site-packages/itk/support/base.py", line 259, in load
    l_spec.loader.exec_module(l_module)  # pytype: disable=attribute-error
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/user/py_env/lib/python3.7/site-packages/itk/support/../TextureFeaturesPython.py", line 74, in <module>
    from itk.itkCoocurrenceTextureFeaturesImageFilterPython import *
  File "/home/user/py_env/lib/python3.7/site-packages/itk/itkCoocurrenceTextureFeaturesImageFilterPython.py", line 3563, in <module>
    from itk.support import itkHelpers
ImportError: cannot import name 'itkHelpers' from 'itk.support' (unknown location)
```

Just updated itk package from pip to version 5.2 and I cannot run itk-texturefeatures anymore, here it's the Traceback for the example code found on this repository.

Searching a bit, I found [this commit from ITK repo](https://github.com/InsightSoftwareConsortium/ITK/commit/ed11fdcf7321512641eaef55ecc60cb9919d3c7e) which deleted `filter_args(object)` from `itk.support.helpers` which also is referenced on `itkCoocurrenceTextureFeaturesImageFilterPython.py` as `from itk.support import itkHelpers`. So it was renamed from `itkHelpers` to `helpers`.

Also, `accept_numpy_array_like_xarray()` was changed to `accept_array_like_xarray_torch()`

[helpers.py source file](https://github.com/InsightSoftwareConsortium/ITK/blob/6dfbc9f0314b5fd64d4b2ae392ae3cc6dda17677/Wrapping/Generators/Python/itk/support/helpers.py)

Hope this information somehow help.

**Comment by lucasjome** [comment]
Forgot to mention that, by replacing:
- `itkHelpers` with `helpers`
- `accept_numpy_array_like_xarray` with `accept_array_like_xarray_torch`
- `itkTemplate` with `template_class`

on bothfiles, `itkCoocurrenceTextureFeaturesImageFilterPython.py` and  `itkRunLengthTextureFeaturesImageFilterPython.py`

Does work as a temporary workaround.

**Comment by thewtex** [comment]
Thanks for the report @lucasjome . Will be addressed by #92

**Comment by thewtex** [comment]
After upgrading to `itk-texturefeatures` `3.5.0` with `pip install --upgrade itk-texturefeatures`, and the issue persists, please re-open -- thanks!
